tests/test_pagination_tag.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from django.test.client import Client
+from wagtail.core.models import Page
+from wagtailnhsukfrontend.templatetags.nhsukfrontend_tags import pagination
+
+
+client = Client()
+
+
+def get_pagination_context(page, request):
+    """Get the pagination context that will be passed to the pagination template."""
+    fake_context = {"page": page, "request": request}
+    # The pagination tag is an inclusion_tag which returns a new context.
+    new_context = pagination(fake_context)
+    return new_context
+
+
+def get_pagination_page(number):
```

Comment:
This function is called `get_pagination_page` but it returns a context, not a page.